src/EncounterPage/EncounterPage.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import EnemyEntry from './EnemyEntry'
+import PlayerEntry from './PlayerEntry'
+
+describe('EncounterPage', () => {
+  it('renders correctly', () => {
+    const tree = renderer.create(
+      <>
+        <EnemyEntry enemy={'Enemy 1'} maxHP={50} ac={17} />
```

Comment:
What is the prop ac?

Reply:
Armor-Class. Its a stat in the game, and in an encounter-situation it is very handy to see it at a glance.

Reply:
I understand your point, but if you would give me the assignment to change the armor class, I would be stuck at first. I don't know easy that behind ac is amor class. After some time I might find out what ac means, or I come back to you and ask you where to find it. If in the code the naming was amorClass, this would not be happened, because the variable name was clear.

Reply:
Thanks for your input! It makes sense to be as clear as possible so that other contributors can easily understand what I mean. I just thought since I mention what AC stands for in the corresponding user stories that it wouldn't be as much of a problem, but I'll change it to armorClass in the future to be safe.

Reply:
My bad, I had read over this :-( 
But future martin will not have the user stories to understand what ac is ;-)
thx for considering to change the naming.